crates/net/network/src/eth_requests.rs

```diff
@@ -144,6 +152,11 @@ where
             }
         }
 
+        // TODO: remove this once we deprecated l2geth
```

Comment:
Can we open an issue and add permalinks to all `TODO` items that we need to address once we have deprecated l2geth please?